external-api/src/constants/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './deal.constant';
 export * from './emailTemplates.constant';
 export * from './amendment.constant';
 export * from './errors.constant';
+export * from './mdm.constant';
```

Comment:
Might be beneficial to move to `libs/common` directory.

Reply:
At the moment it has just one value and it is used just by external API.